src/_mixins.scss

```diff
@@ -20,7 +20,7 @@
   #{$property}: get-width($columns, $position, get-multiplier-unit($position));
 }
 
-/// Set the offset width for the provided property based on the column count
+/// Set the offsets width for the provided property based on the column count
```

Comment:
Same issue with possessiveness. I recommend "Set the width of offsets for…" instead.